Content.Server/Singularity/Events/EventHorizonConsumedEntityEvent.cs

```diff
@@ -4,7 +4,7 @@
 namespace Content.Server.Singularity.Events;
 
 /// <summary>
-/// Event raised on the event horizon entity whenever an event horizon consumes an entity.
+/// Event raised on the entity being consumed whenever an event horizon consumes an entity.
 /// </summary>
 public sealed class EventHorizonConsumedEntityEvent : EntityEventArgs
 {
```

Comment:
This needs to be a struct event because it's gonna get spammed out.

Reply:
Alright.